src/renderer/SiteInfoMenuItem.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { useStoreSelector, selectors } from './store/store';
+import {
+	Spinner,
+} from '@getflywheel/local-components';
+import { OptimizerStatus } from '../types';
+
+const SiteInfoMenuItem: React.FC = () => {
+	const selectedSiteImages = useStoreSelector(selectors.selectedSiteImages);
+
+	const selectedImages = useStoreSelector(selectors.selectedSiteImages);
```

Comment:
Looks like there is a duplicate `useStoreSelector(selectors.selectedSiteImages);` here